team/guide/issue-triage.md

```diff
@@ -8,65 +8,41 @@ editor: markdown
-- One `status:` label describing the current status of the issue. If the issue has been triaged, this should not be `status: triage`.
+1. **Set optional labels**
+  
+    Scan through the full labels list and apply other labels as necessary. Learn more about the [meaning of all our labels](https://github.com/centerofci/mathesar/labels).
 
-There are also other labels you should consider applying:
-- `priority` labels for issues that are either urgent or are for future consideration.
-- `restricted:` labels for issues that are not open to the community.
-  - All `work: design` issues should also be marked `restricted: design team` because only the design team can currently work on those.
-  - Consider marking complicated issues or issues that touch the fundamental architecture of Mathesar as `restricted: maintainers`.
-- `good first issue` and `help wanted` labels for issues that are particularly suitable for community contribution.
-  - Ensure that `help wanted` is applied to all issues of this type, but only apply `good first issue` in addition to `help wanted` if the issue is approachable for new and inexperienced contributors.
-- `affects:` labels for issues that touch technical debt, architecture, DX, or UX.
-- Any other labels that apply from our [repository's configured labels](https://github.com/centerofci/mathesar/labels).
 
-Applying the correct label should also sort the issue correctly in the Mathesar GitHub project.
+1. **Verify Project and fields**
 
-### Ensure the issue is sorted correctly in the Mathesar project
+    Set the Project to "Mathesar".
 
-The [Mathesar GitHub project](https://github.com/orgs/centerofci/projects/1) organizes our open issues. We need to ensure that the issue is in the project and the following fields are set correctly.
-- Status
-- Priority
-- Work
+    Within the project, ensure that the `status`, `priority`, and `work` fields are set.
 
-This process should be fully automated based on setting the correct labels in the previous step.
+    > The project is only accessible to [Team](/team) members, but all relevant information (such as status, priority, and milestone) should be available on individual issues. We will make this project public as soon as GitHub supports it.
+    {.is-warning}
 
-> The project is only accessible to [Team](/team) members, but all relevant information (such as status, priority, and milestone) should be available on individual issues. We will make this project public as soon as GitHub supports it.
-{.is-warning}
+1. **Set milestone**
 
-### Assigning milestones
-All issues should have a milestone.
+    If an issue is directly associated with a feature, put it in the milestone for that feature. Otherwise, put it in a monthly improvement milestone (named like `YYYY-MD improvements`) according to priority. More urgent issues should go in this month's milestone, otherwise put it in a milestone in the next couple of months.
```

Comment:
I renamed these to "General Improvements" milestones, this should reflect that.

Reply:
Fixed in 77d66b923fdaa5d5d39285f199ed54641d9ed76a